pymc/sampling/forward.py

```diff
@@ -829,7 +829,9 @@ def sample_posterior_predictive(
     _log.info(f"Sampling: {list(sorted(volatile_basic_rvs, key=lambda var: var.name))}")  # type: ignore
     ppc_trace_t = _DefaultTrace(samples)
     try:
-        with Progress(console=Console(theme=progressbar_theme)) as progress:
+        with Progress(
+            console=Console(theme=progressbar_theme), disable=not progressbar
+        ) as progress:
```

Comment:
We should do this everywhere we use the Progressbar no? `sample`, `smc` and here: https://github.com/pymc-devs/pymc/blob/6761c0c73ba07cc9dc51ec3adab7f1aa5f76b23d/pymc/backends/arviz.py#L662

Not sure if I am missing any other place